tests/envs/test_narrow_passage.py

```diff
@@ -0,0 +1,287 @@
+
+    for action in door_open_actions:
+        s = env.simulate(s, Action(action))
+    # check door is open
+    assert s.get(door, "open") == 1
+    assert GroundAtom(DoorIsOpen, [door]).holds(s)
+
+    for action in move_target_actions:
+        s = env.simulate(s, Action(action))
+    assert GroundAtom(TouchedGoal,
+                      [robot, target]).holds(s)  # check touching goal
+    assert task.goal_holds(s)  # check task goal reached
+
+    # Test rendering entire plan
+    policy = utils.action_arrs_to_policy(all_action_arrs)
+    traj = utils.run_policy_with_simulator(policy,
+                                           env.simulate,
+                                           task.init,
+                                           task.goal_holds,
+                                           max_num_steps=len(all_action_arrs))
+
+    # Render a state before door opens
+    env.render_state(traj.states[2], task)
+    # Render a state after door opens
+    env.render_state(traj.states[4], task, caption="caption")
+    # Render state at end of trajectory
+    env.render_state(traj.states[-1], task)
+
+
+def test_narrow_passage_collisions():
+    """Test that the robot can't go through walls or closed door."""
+    # set up environment
+    utils.reset_config({"env": "narrow_passage"})
+    env = NarrowPassageEnv()
+    door_type, _, robot_type, _, _ = sorted(env.types)
+
+    # Test robot should not be able to walk through each wall nor the door
+    sample_task = env.get_train_tasks()[0]
+    test_robot_xs = [0.1, 0.35, 0.7, 0.9]
+    y_midpoint = env.y_lb + (env.y_ub - env.y_lb) / 2
+    down_action = Action(np.array([0, -0.08, 0]).astype(np.float32))
+    for test_x in test_robot_xs:
+        state = sample_task.init.copy()
+        robot, = state.get_objects(robot_type)
+        state.set(robot, "x", test_x)
+        # try going down a lot
+        for _ in range(50):
+            state = env.simulate(state, down_action)
+        # make sure robot is still above the wall
+        robot_y = state.get(robot, "y")
+        assert robot_y > y_midpoint, f"Robot did not collide at x={test_x}"
+
+    # Test robot can walk through the doorway when the door is open
+    state = sample_task.init.copy()
+    robot, = state.get_objects(robot_type)
+    door, = state.get_objects(door_type)
+    state.set(robot, "x", 0.35)
+    state.set(door, "open", 1)  # set the door to be open
+    # try going down a lot
+    for _ in range(50):
+        state = env.simulate(state, down_action)
+    # make sure robot is below the wall now but not out of bounds
+    robot_y = state.get(robot, "y")
+    assert robot_y < y_midpoint, "Robot wasn't able to pass through doorway"
+    assert robot_y >= env.y_lb, "Robot went out of bounds"
+
+
+def test_narrow_passage_options():
+    """Test ParameterizedOptions work."""
+    # Set up environment
+    utils.reset_config({
+        "env": "narrow_passage",
+        "render_state_dpi": 150,
```

Comment:
any reason to keep this high in the tests? we want the tests to be as fast as possible, so the low default value is preferred